community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -562,12 +567,14 @@
         AUTO
     }
 
+    /**
+     * @deprecated - lucene label index has been removed, thus 'native' is only viable option and this setting is not needed.
+     */
+    @Deprecated
     @Description( "Backend to use for label --> nodes index" )
     @Internal
-    public static final Setting<String> label_index = setting( "dbms.label_index",
-            options( enumNames( LabelIndex.class ), true ), LabelIndex.NATIVE.name() );
-
-    // Security settings
+    public static final Setting<String> label_index =
+            setting( "dbms.label_index", optionsIgnoreCase( NATIVE_LABEL_INDEX_TAG ), NATIVE_LABEL_INDEX_TAG );
```

Comment:
Legacy, forgot to change that back after I reverted removal of `LabelIndex` enum. Fixing!

Hmm, my initial thinking was to remove it because it "has no meaning" anymore. However you could argue that whoever is using this AUTO setting "just want stuff to work", which is fair. So I agree, allow AUTO seems to be the least intrusive thing to do.